docs/getting-started.md

```diff
@@ -0,0 +1,325 @@
+# Getting started with JupyterHub
+
+This document describes some of the basics of configuring JupyterHub to do what you want.
+JupyterHub is highly customizable, so there's a lot to cover.
+
+
+## Installation
+
+See [the readme](../README.md) for help installing JupyterHub.
+
+
+## JupyterHub's default behavior
+
+Let's start by describing what happens when you type `sudo jupyterhub`
+after installing it, without any configuration.
+
+
+### Authentication
+
+The default Authenticator that ships with JupyterHub
+authenticates users with their system name and password (via [PAM][]).
+Any user on the system with a password will be allowed to start a notebook server.
+
+
+### Spawning servers
+
+The default Spawner starts servers locally as each user,
+one for each server. These servers listen on localhost,
+and start in the given user's home directory.
```

Comment:
Ah I see you mention that this is configurable below in the `Configuring single-user servers` section.  You probably should reference that.